pkg/handlers/nutanix/mutation/machinedetails/inject.go

```diff
@@ -92,9 +92,9 @@ func (h *nutanixMachineDetailsPatchHandler) Mutate(
 
 			spec := obj.Spec.Template.Spec
 
-			spec.BootType = capxv1.NutanixBootType(nutanixMachineDetailsVar.BootType)
-			spec.Cluster = capxv1.NutanixResourceIdentifier(nutanixMachineDetailsVar.Cluster)
-			spec.Image = capxv1.NutanixResourceIdentifier(nutanixMachineDetailsVar.Image)
+			spec.BootType = nutanixMachineDetailsVar.BootType
```

Comment:
Thanks! I think I missed it as part of https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/698?

Reply:
Yes, don't miss this again @dkoshkin 😛